blueocean-dashboard/src/main/js/AllPipelines.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import OrganisationPipelines from './OrganisationPipelines';
+
+class AllPipelines extends Component {
+
+    render() {
+        return (
+            <OrganisationPipelines { ... this.props } />
```

Comment:
I noticed the REST API is spelled like "organization" so I wanted to be consistent with that. Will do some renaming before this PR is done to get it all consistent.